lib/components/__tests__/ArticleList.js

```diff
@@ -18,14 +18,14 @@ describe('ArticleList', () => {
 	it('render correctly', () => {
 		const wrapper = shallow(
 			<ArticleList
-			    { ...testProps }
+			    {...testProps}
 			/>
 		);
 		
-		//console.log(wrapper.node.props.children);
+		console.log(wrapper.node.props.children);

```

Comment:
Make sure to comment or remove the console statement still failing test

Reply:
ohhhhhh it is mine bad